README.md

```diff
@@ -41,74 +111,85 @@ The project is available in two ways and should be consumed accordingly:
 ```
 
 ## Storybook
 
 Spin up storybook by running this command in the terminal:
 
-```bash
+```shell
 npm run storybook
 ```
 
-When storybook is ready it automatically opens up in a browser with the interface ready to use.
+When storybook is ready it automatically opens up in a browser with the
+interface ready to use.
 
 ## Chromatic
 
 We are using Chromatic for visual test. You can access the dashboard
-under the `danskernesdigitalebibliotek` (organisation) `dpl-design-system` (project).
+under the `danskernesdigitalebibliotek` (organisation) `dpl-design-system`
+(project).
 
-https://www.chromatic.com/builds?appId=616ffdab9acbf5003ad5fd2b
+<https://www.chromatic.com/builds?appId=616ffdab9acbf5003ad5fd2b>
 
 You can deploy a version locally to Chromatic by running:
 
-```bash
+```shell
 npm run chromatic
 ```
 
-Make sure to set the `CHROMATIC_PROJECT_TOKEN` environment variable is available in your shell context.
-You can access the token from:
+Make sure to set the `CHROMATIC_PROJECT_TOKEN` environment variable is available
+in your shell context. You can access the token from:
 
-https://www.chromatic.com/manage?appId=616ffdab9acbf5003ad5fd2b&view=configure
+<https://www.chromatic.com/manage?appId=616ffdab9acbf5003ad5fd2b&view=configure>
 
 ### What is Storybook
 
-[Storybook](https://storybook.js.org/docs/react/get-started/introduction) is an open source tool for building UI components and pages in isolation from your app's business logic, data, and context.
-Storybook helps you document components for reuse and automatically visually test your components to prevent bugs.
-It promotes the [component-driven](https://componentdriven.org) process and agile development.
+[Storybook](https://storybook.js.org/docs/react/get-started/introduction) is an
+open source tool for building UI components and pages in isolation from your
+app's business logic, data, and context. Storybook helps you document components
+for reuse and automatically visually test your components to prevent bugs. It
+promotes the [component-driven](https://componentdriven.org) process and agile
+development.
 
-It is possible to extend Storybook with an ecosystem of addons that help you do things like fine-tune responsive layouts or verify accessibility.
+It is possible to extend Storybook with an ecosystem of addons that help you do
+things like fine-tune responsive layouts or verify accessibility.
 
 #### How to use
 
-The Storybook interface is simple and intuitive to use. Browse the project's stories now by navigating to them in the sidebar.
-The stories are divided into "Components" and "Atoms" where atoms are the smaller building blocks that can be used to build bigger components.
-Each Atom/Component has a corresponding `.stories` file. View their code in the `src/stories` directory to learn how they work.
-The `stories` file is used to add the component to the Storybook interface.
+The Storybook interface is simple and intuitive to use. Browse the project's
```

Comment:
I would remove this line "The Storybook interface is simple and intuitive to use."